controllers/protectedController.js

```diff
@@ -0,0 +1,19 @@
+import { ProtectedRepository } from "../repositories/protectedRepository.js";
+
+export class ProtectedController {
+  constructor() {
+    this.protectedRepository = new ProtectedRepository();
+  }
+
+  getProtected = async (req, res, next) => {
+    console.log("getProtected")
+
+    try {
+      const response = await this.protectedRepository.getProtected();
+      res.status(666).json(response);
+    } catch (error) {
+      res.status(401).json({ message: "please send auth headers" });
```

Comment:
I would recommend sending error code instead of the error message. Something like : `Unathorized `or `AuthHeadersMissing`. It would be much easier for the frontend to handle